examples/inverse/plot_tf_lcmv.py

```diff
@@ -41,10 +41,11 @@
 # Pick a selection of magnetometer channels. A subset of all channels was used
 # to speed up the example. For a solution based on all MEG channels use
 # meg=True, selection=None and add grad=4000e-13 to the reject dictionary.
+# We could do this with a "picks" argument to Epochs and the LCMV functions,
+# but here we use raw.pick_types() to save memory.
```

Comment:
does proj=True in raw? Given the pick / proj PR mess I think we should. I am curious if it changes the source locations.